docs/source/documentation/concepts.rst

```diff
@@ -43,19 +43,20 @@ Compute plan and tasks
 
 .. _concept_compute_plan:
 
-Compute plan
-""""""""""""
+**Compute plan**
+^^^^^^^^^^^^^^^^
+
 A set of tasks.
 Gathering tasks into a single compute plan will lead to a more optimized compute.
 
 Note that you can register a task alone, i.e. not put the task in a compute plan, but Substra will still create a compute plan for you for this specific task.
 
-Task
-""""
+**Task**
```

Comment:
Why did you change the title format to bold ?

Reply:
It is hard to see that these are titles/headings because the text is almost the same size. Wasn't clear to me as I went through the documentation @ThibaultFy 
https://share.getcloudapp.com/geuOoe7P

Reply:
Ok I see, there is a problem indeed, I'll make some suggestion to keep them a rst title